asciidoc-common/install/proc-Installing_the_Websocket_Proxy_on_a_different_host.adoc

```diff
@@ -0,0 +1,133 @@
+[id='Installing_the_Websocket_Proxy_on_a_different_host_{context}']
+= Installing a Websocket Proxy on a Separate Machine
+ifdef::rhv-doc[]
+[IMPORTANT]
+====
+The websocket proxy and noVNC are Technology Preview features only. Technology Preview features are not supported with Red Hat production service-level agreements (SLAs) and might not be functionally complete, and Red Hat does not recommend using them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information see link:https://access.redhat.com/support/offerings/techpreview/[Red Hat Technology Preview Features Support Scope].
+====
+endif::[]
+
+The websocket proxy allows users to connect to virtual machines through a noVNC console. The noVNC client uses websockets to pass VNC data. However, the VNC server in QEMU does not provide websocket support, so a websocket proxy must be placed between the client and the VNC server. The proxy can run on any machine that has access to the network, including the the {engine-name} machine.
+
+For security and performance reasons, users may want to configure the websocket proxy on a separate machine.
+
+.Procedure
+
+. Install the websocket proxy:
++
+[options="nowrap" subs="normal"]
+----
+# yum install ovirt-engine-websocket-proxy
+----
++
+. Run the `engine-setup` command to configure the websocket proxy.
++
+[options="nowrap" subs="normal"]
+----
+# engine-setup
+----
++
+[NOTE]
+====
+If the `rhvm` package has also been installed, choose `No` when asked to configure the {engine-name} (`Engine`) on this host.
```

Comment:
this note should be on rhv doc, on ovirt we have ovirt-engine instead.

Reply:
Added a conditionalized note for ovirt.